### DISCUSSION TITLE
Meet uWidgets

### DISCUSSION THREAD

**Discussion by creewick** [creewick/MontereyRainmeter#106] [Announcements]
Dear MontereyRainmeter Users,

I know, this repository was inactive for a while, but I still have a lot of ideas related to it.
Unfortunately, not all of these ideas is possible to implement with the Rainmeter engine.
As mentioned previously, I'm working on another version of the widget pack, with a custom .NET engine.

I will share my progress with you on another repository:
https://github.com/creewick/uWidgets

Right now, the early beta version is available to download. It doesn't contain all the widgets from MontereyRainmeter, and may be not as stable. But I'll be glad if you give it a try and share your feedback.

I also want to say, that all the issues and suggestions you have published on this repository will be taken into account during the development of Widgets.

Thank you for your support and understanding. I hope you enjoy using uWidgets

**Comment by jtlw99** [comment]
even better
